rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1927,41 +1886,21 @@ void settings_dialog::OnBackupCurrentGuiConfig()
 			QMessageBox::warning(this, tr("Error", "Backup GUI config warning 2"), tr("Must choose a name with no '.'", "Backup GUI config warning 2"));
 			continue;
 		}
-		if (ui->combo_configs->findText(gui_config_name) != -1)
-		{
-			QMessageBox::warning(this, tr("Error", "Backup GUI config warning 3"), tr("Please choose a non-existing name", "Backup GUI config warning 3"));
-			continue;
-		}
 		Q_EMIT GuiSettingsSaveRequest();
 		m_gui_settings->SaveCurrentConfig(gui_config_name);
-		ui->combo_configs->addItem(gui_config_name);
-		ui->combo_configs->setCurrentText(gui_config_name);
 		m_current_gui_config = gui_config_name;
 		break;
 	}
 }
 
 void settings_dialog::OnApplyGuiConfig()
 {
-	const QString new_config = ui->combo_configs->currentText();
 
-	if (new_config == m_current_gui_config)
-	{
-		return;
-	}
 
 	// Backup current window states
 	Q_EMIT GuiSettingsSaveRequest();
 
-	if (!m_gui_settings->ChangeToConfig(new_config))
-	{
-		const int new_config_idx = ui->combo_configs->currentIndex();
-		ui->combo_configs->setCurrentText(m_current_gui_config);
-		ui->combo_configs->removeItem(new_config_idx);
-		return;
```

Comment:
Why not `return const` here? That would probably fix the bloom issue with Killzone Shadow Fall

Reply:
The AI takes care of it.